uiowa/uiowa
# Initial platform email setup.

Related to #3498 

### Testing
- Go to https://default.dev.drupal.uiowa.edu/admin/config/development/reroute_email and temporarily disable email rerouting.
- Go to https://default.dev.drupal.uiowa.edu/email-test and submit the form with different variations:
  - From a uiowa.edu address with a from name provided.
  - From a uiowa.edu address with no from name provided.
  - From a non-uiowa.edu address with a from name provided.
  - From a non-uiowa.edu address with no from name provided.
  - To multiple email addresses.
- Check your inbox for the messages and inspect them.
- Reply to the messages and inspect the reply-to address.
- Test brand dev according to https://github.com/uiowa/uiowa/pull/4619#issuecomment-1006822212
- Test D7: https://org-iowareadingresearch.dev.drupal.uiowa.edu/contact-us

Feel free to reconfigure the dev webform to test other email functionality - cc, bcc, etc. Note that multiple from addresses do not currently work, even with legacy email in prod.
